Classes/Helper/FormDataReader.php

```diff
@@ -261,8 +261,11 @@ protected function getNewAndUpdatedFiles()
 
     public function uploadError()
     {
-
-        if ($this->formData['primaryFile'] && $this->formData['primaryFile']['error'] != 0) {
+        if (
+            $this->formData['primaryFile'] &&
+            $this->formData['primaryFile']['error'] != 0 &&
```

Comment:
Magic numbers for primary file errors. What is the meaning?

Reply:
The numbers have been replaced by php constants for file upload errors.